src/convertToObject.js

```diff
@@ -5,8 +5,61 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
+function convertToObject(
+  sourceString = `
+background-color:      #fff;
+-webkit-border-radius: 5px;
+border-radius     : 5px;
+border: 1px solid #e8e8e8;
+-webkit-box-sizing: border-box;
+box-sizing: border-box;
+clear   : both       ;
+cursor: pointer;
+float: left;
+font-family: inherit;
+    font-size: 14px;
+font-weight: 400;
+height: 42px;
+line-height:    40px;
+outline: 0;
+padding-left    : 18px;
+padding-right: 30px;
+;
+
+;
+position: relative;
+
+
+text-align: left !important;
+-webkit-transition: all .2s ease-in-out;
+transition: all .2s ease-in-out;
+-moz-user-select: none;
+-ms-user-select: none;
+user-select: none;
+
+
+white-space: nowrap;
+width: auto;
+`,
+) {
   // write your code here
+  const arr = sourceString.split(';');
+
+  arr.forEach((el) => el.trim());
```

Comment:
The `forEach` method does not modify the array in place. The callback function should perform an action with side effects or the result should be assigned to a new array. As it stands, this line has no effect.

Reply:
not fixed

it is better to use map in this case